.github/workflows/pr-build.yaml

```diff
@@ -235,6 +235,7 @@ jobs:
             -DskipTests \
             -pl :camel-quarkus-integration-test-base64 \
             -pl :camel-quarkus-integration-test-bindy \
+            -pl :camel-quarkus-integration-test-gson \
```

Comment:
wonder if we can merge gson and jackson tests in a single dataformat-json module (similar to what we do for http)